src/main/java/ch/njol/skript/conditions/CondAI.java

```diff
@@ -0,0 +1,31 @@
+package ch.njol.skript.conditions;
```

Comment:
You forgot the copyright thingy at the top of your new classes

Reply:
oops thanks